recipes/REAL-M/sisnr-estimation/train.py

```diff
@@ -158,7 +158,7 @@ def compute_oracle_sisnrs(self, predictions, targets):
     def fit_batch(self, batch):
         """Trains one batch"""
 
-        if self.hparams.use_whamr_train:
+        if self.hparams.use_whamr_train and self.hparams.dynamic_mixing:
```

Comment:
Important change here, where does it come from?

Reply:
It was failing for some reason unless this was in - don't remember. 

@ycemsubakan please take a look & make a suggestion what needs to happen.

My concern was that testing runs through - this seemed like a somewhat plausible fix (w/o expert knowledge being available).